### DISCUSSION TITLE
[Bug]: get directions asks to allow or deny and fails

### DISCUSSION THREAD

**Discussion by furick1** [code4recovery/12-step-meeting-list#1378] [Q&A]
### Contact Details

Furick1@gmail.com

### Website With Issue

https://recoverydharma.org/meetings/red-deer-recovery-community-dharma/

### What happened?

The get directions link pops an allow or deny modal on iPhone but doesn’t follow thru to google maps. Could this be some conflict with mapbox or an iOS update breaking a prior functioning feature?

### What browsers are you seeing the problem on?

Chrome

**Comment by joshreisner** [comment]
hm, might be a problem with the plugin and chrome for ios - works ok for you on safari?

**Reply by joshreisner** [reply]
(moved this to a discussion)

hm, that link works ok for me! did you uninstall the iOS maps app by chance?

**Reply by joshreisner** [reply]
wondering if this is related to https://github.com/code4recovery/12-step-meeting-list/discussions/1369 - @furick1 when you get a chance could you confirm whether you have the iOS maps app installed? thanks

**Comment by furick1** [comment]
Same on safari

**Comment by furick1** [comment]
I uninstalled Apple and Google maps and reinstalled. Now Apple Maps will launch from safari but it’s still broken on chrome on iPhone 13 Pro running latest iOS of 17.3.1.
